### PR TITLE
try to run tests with ats

### PR DESCRIPTION
**Pull request by ubergesundheit**
Changes for the upcoming split of app-build-suite and app-test-suite in ci

### Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by ubergesundheit** [comment]
@MarcelMue feel free to hijack this

**Comment by ubergesundheit** [comment]
yo @stone-z check this out

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,8 +2,22 @@
...
+      ats_version:
+        type: string
+        default: 0.1.4
+      chart_archive_prefix:
+        type: string
+    steps:
+      - architect/run-tests-with-ats:
+          chart_archive_prefix: "<< parameters.chart_archive_prefix >>"
+          app-test-suite_version: "v<< parameters.ats_version >>"
+          app-test-suite_container_tag: "<< parameters.ats_version >>"
```
Comment (rossf7) [inline .circleci/config.yml:19]: I'm a bit triggered by the hypens and underscores here :) but it makes sense.

Same for ordering the parameters but there aren't that many.

**Review by rossf7 — APPROVED** [review]
LGTM

File: .circleci/config.yml (hunk 1 above)
Comment (MarcelMue) [reply .circleci/config.yml:19]: I think we use underscores all over the place in architect-orb as well. Not really sure why :sweat_smile:

**Comment by ubergesundheit** [comment]
want to do any changes or should I merge?

**Comment by MarcelMue** [comment]
PRESS THE BUTTON

**Pull request merged**